server/scripts/import-tests/index.js

```diff
@@ -313,27 +394,219 @@ const getVersionString = async ({ directory, updatedAt }) => {
+
+        // Using the v1 test format, https://github.com/w3c/aria-at/wiki/Test-Format-V1-Definition
+        if (!isV2) {
+            const common = allCollected[0];
+            const testId = createTestId(testPlanVersionId, common.info.testId);
+            const atIds = allCollected.map(
+                collected =>
+                    ats.find(at => at.name === collected.target.at.name).id
+            );
+
+            tests.push({
+                id: testId,
+                rowNumber: rawTestId,
+                title: common.info.title,
+                atIds,
+                atMode: common.target.mode.toUpperCase(),
+                renderableContent: Object.fromEntries(
+                    allCollected.map((collected, index) => {
+                        return [atIds[index], collected];
+                    })
+                ),
+                renderedUrls: Object.fromEntries(
+                    atIds.map((atId, index) => {
+                        return [atId, getRenderedUrl(index)];
+                    })
+                ),
+                scenarios: (() => {
+                    const scenarios = [];
+                    allCollected.forEach(collected => {
+                        collected.commands.forEach(command => {
+                            scenarios.push({
+                                id: createScenarioId(testId, scenarios.length),
+                                atId: ats.find(
+                                    at => at.name === collected.target.at.name
+                                ).id,
+                                commandIds: command.keypresses.map(
+                                    ({ id }) => id
+                                )
+                            });
+                        });
+                    });
+                    return scenarios;
+                })(),
+                assertions: getAssertions(common, testId),
+                viewers: []
+            });
+        }
+
+        // Using the v2 test format, https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2
+        if (isV2) {
+            for (const [collectedIndex, collected] of allCollected.entries()) {
+                const testId = createTestId(
+                    testPlanVersionId,
+                    `${collected.target.at.key}:${collected.info.testId}`
+                );
+
+                let test = {
+                    id: testId,
+                    rawTestId,
+                    rowNumber: collected.info.presentationNumber,
+                    title: collected.info.title,
+                    at: {
+                        key: collected.target.at.key,
+                        name: collected.target.at.name,
+                        settings: collected.target.at.raw.settings
+                    },
+                    atIds: [
+                        ats.find(at => at.name === collected.target.at.name).id
+                    ],
+                    renderableContent: {
```

Comment:
> I made a comment in one of your other PRs that I thought it made sense to add a testFormatVersion field to the renderableContent and then allowing the differences in the test format to be faithfully expressed within that section. What do you think?

To me, that then puts work on a resolver (or some other server op) to evaluate some change in the `renderableContent` based on that `testFormatVersion`. I'm not against it, but I am a bit lost, on why add to each `test.renderableContent` rather than referencing the parent `TestPlanVersion.metadata`.

> renderableContent was to faithfully preserve the collected.json file content

A big thing about this change was also maintaining the already existing structure of `renderableContent` to avoid unnecessary hiccups and maintaining that control. But the majority of `renderableContent` is the same as the collected file, and could have been `collected` or `{ ...collected, extras }` really. Outside of excluding `target.ats` which the new collected.json would have, and also making changes to assertions to use [tokenized assertionStatements](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#assertionstatement) where needed, it is exactly the same.

I can make a change to reflect that.